sdk/src/index.d.ts

```diff
@@ -29,7 +46,7 @@ export {
 	Application,
 	version,
 	systemDirs,
-	configurator,
+	Configurator,
```

Comment:
this configurator is not a class. it was exporting default configs

Reply:
So it is an instance of `Configurator`?

Reply:
yes!